src/redux/selectors/userSelector.js

```diff
@@ -0,0 +1,14 @@
+import { createSelector } from 'reselect'
+
+export const isWalletValid = createSelector(
+  state => state.currentUser.wallet_valid_until,
+  wallet_valid_until => {
+    const number = Date.now()
+
+    if (wallet_valid_until != null) {
+      return number < new Date(wallet_valid_until)
```

Comment:
Je ne sais pas si `number` est très parlant. :)

```suggestion
    const number = Date.now()

    if (wallet_valid_until != null) {
      return new Date(wallet_valid_until) > Date.now()
```

Au passage, je renverse la condition, ça me semble plus naturel à lire : "est-ce que c'est valide après 'now'".